### PR TITLE
Fix home dom

### PR DESCRIPTION
**Pull request by aminalakhsham**
Relates #62 edit pop up in dom
Relates #63 edit css file for home page

### CODE REVIEW

File: public/js/selectorFuction.js [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
-  enterTypeofQuery.map(
-    (e, i) => {
-      elements[selectorsName[i]] = document.querySelector(e);
-      return '';
-    },
-  );
+  enterTypeofQuery.map((e, i) => {
+    elements[selectorsName[i]] = document.querySelector(e);
+    return '';
+  });
```
Comment (tomduggan85) [inline public/js/selectorFuction.js:8]: @aminalakhsham I just noticed that you aren't using the return value of this map, so you could change it to a `forEach` and then you could remove the `return '';` Not super important though 🤷‍♂️

File: test/logic.test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+test('test Funaction for validation email(wrong email)', (t) => {
+  t.equal(validateEamil('amin'), false, 'email not validate');
+  t.end();
+});
+
+test('test Funaction for validation email(correct email)', (t) => {
+  t.equal(validateEamil('amin@gmail.com'), true, 'email validate');
+  t.end();
+});
```
Comment (tomduggan85) [inline test/logic.test.js:12]: @aminalakhsham  Nice job adding tests!

**Comment by tomduggan85** [comment]
@aminalakhsham thanks for making these changes!

**Review by InassTubail — APPROVED** [review]

**Comment by aminalakhsham** [comment]
i love your comment 
thank you @tomduggan85

**Comment by tomduggan85** [comment]
@aminalakhsham you’re welcome!

**Pull request merged**